okteto-pipeline.yml

```diff
@@ -1,7 +1,7 @@
 icon: https://github.com/okteto/chat/raw/master/icon.png
 deploy:
-  - kubectl apply -f dev
   - okteto build -t okteto.dev/chat:$OKTETO_GIT_COMMIT
+  - kubectl apply -f dev
```

Comment:
It is better to let the database start while the image is being deployed

Reply:
@pchico83 why is that? I did it this way because build the image takes a long time, and it's an odd experience to have the db in ready state for so long while the image is still building.

Reply:
why it is odd? it is better to do things in parallel if they don't depend on each other. running the database first ensures the db is ready when the app starts.

Reply:
It's a good point, but it looks odd in the UI. the feedback we give users while the image still has a lot of long pauses where it feels like things are stuck. When that happens, seeing some resources with the 'available' state while the pipeline is still in progress is still in progress creates a lot of noise to me. Specially because the rest of the resources only appear until after we hit the helm upgrade phase. 

I think that launching the DB, Redis and the app together could give a better experience. otherwise, since at that point you see the full list of resources in progress.  Instead, you see a progressiong of of two greens, 3 minutes of progressing, then 2 green and  2 blue resources, and then finally everything green. It's just odd.

Reply:
We should pipe logs, but that is a different story. But running the db first is the right thing

Reply:
I don't think so. I think deploying all the resources at the same times gives a clearer picture, specially to those who are not familiar with the content of the pipeline. Seeing the status of all resources together makes it easier to understand what's going on, and how much work is left to do, because everything is there, instead of progressively appearing.

Reply:
I use this sample for demos and I like it better this way. Create another sample if you want

Reply:
I deployed your branch and this is what happened:
```
sidekiq-69bd6757c4-d4m52sidekiq2020-10-08T09:47:30.189Z pid=1 tid=gqvt5xmjp INFO: Booting Sidekiq 6.1.1 with redis options {:url=>"redis://redis:6379/", :password=>nil}
sidekiq-69bd6757c4-d4m52sidekiq2020-10-08T09:47:33.385Z pid=1 tid=gqvt5xmjp INFO: Booted Rails 6.0.3.2 application in production environment
sidekiq-69bd6757c4-d4m52sidekiq2020-10-08T09:47:33.385Z pid=1 tid=gqvt5xmjp INFO: Running in ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-linux]
sidekiq-69bd6757c4-d4m52sidekiq2020-10-08T09:47:33.385Z pid=1 tid=gqvt5xmjp INFO: See LICENSE and the LGPL-3.0 for licensing details.
sidekiq-69bd6757c4-d4m52sidekiq2020-10-08T09:47:33.385Z pid=1 tid=gqvt5xmjp INFO: Upgrade to Sidekiq Pro for more features and support: https://sidekiq.org
sidekiq-69bd6757c4-d4m52sidekiq2020-10-08T09:47:43.399Z pid=1 tid=gqvt5xmjp WARN: Redis::CannotConnectError: Error connecting to Redis on redis:6379 (Redis::TimeoutError)
sidekiq-69bd6757c4-d4m52sidekiq2020-10-08T09:47:43.399Z pid=1 tid=gqvt5xmjp WARN: /root/.bundle/ruby/2.6.0/gems/redis-4.2.1/lib/redis/client.rb:362:in `rescue in establish_connection'
```
and the sidekiq service does not try to reconnect